src/EPR.Calculator.Frontend.UnitTests/DashboardControllerTests.cs

```diff
@@ -208,20 +211,9 @@ public void Should_Classify_CalculationRuns_And_Handle_DefaultValue()
 
             // Assert
             Assert.AreEqual(3, dashboardRunData.Count);
-            Assert.AreEqual("IN THE QUEUE", dashboardRunData[0].Status);
-            Assert.AreEqual("RUNNING", dashboardRunData[1].Status);
-            Assert.AreEqual(string.Empty, dashboardRunData[2].Status); // Default value
-        }
-
-        private IConfiguration GetConfigurationValues()
-        {
-            string projectPath = AppDomain.CurrentDomain.BaseDirectory.Split(new string[] { @"bin\" }, StringSplitOptions.None)[0];
-            IConfiguration config = new ConfigurationBuilder()
-               .SetBasePath(projectPath)
-               .AddJsonFile("appsettings.Test.json")
-               .Build();
-
-            return config;
+            Assert.AreEqual(CalculationRunStatus.InTheQueue, dashboardRunData[0].Status);
```

Comment:
```
dashboardRunData[0].Status - dashboardRunData.First().Status

// if this is the last item in the list
dashboardRunData[2].Status - dashboardRunData.Last().Status
```

Reply:
StyleCop is not allowing this, it is asking me to use index instead of first() and last().

Reply:
Thanks Uday, I have updated the code and added the global supression file.